helm-charts/helm2/Makefile

```diff
@@ -0,0 +1,39 @@
+PROJECT_NAME=helm-charts
+
+include ../../Makefile.os
+
+RELEASE_VERSION ?= latest
+CHART_SEMANTIC_RELEASE_VERSION ?= $(shell cat ../../release.version | tr A-Z a-z)
+CHART_NAME=strimzi-kafka-operator
+CHART_PATH ?= ../helm2/strimzi-kafka-operator/
```

Comment:
Inconsistent whitespace–you have it around `?=`, but not `=`.

Reply:
Fixed.